README.md

```diff
@@ -16,3 +16,53 @@ The available endpoints are:
 - `/blogs/new`
 - `/posts/<string:blog_id>`
 - `/posts/new/<string:blog_id>`
+
+
+## HOW TO USE THIS CODE
+
+1) Create a new virtual environment: `python -m venv venv`
+2) Activate the virtual environment:
+
+In powershell:
+
+```powershell
+.\venv\Scripts\activate
+```
+
+In bash:
+
+```bash
+source venv/Scripts/activate
+```
+
+In CMD
+
+```cmd
+venv\Scripts\activate
+```
```

Comment:
Can we include a Mac OS / Linux version?

It's:

```
source .venv/bin/activate
```

Reply:
Added